dns/bind/src/opnsense/service/conf/actions.d/actions_bind.conf

```diff
@@ -22,6 +22,12 @@ parameters:
 type:script_output
 message:request BIND status
 
+[reload]
+command:/usr/local/etc/rc.d/named status && /usr/local/etc/rc.d/named reload
```

Comment:
Why do you need `status`?

Reply:
Because it's stalling if bind is not running.

Reply:
Not needed, like i said it is my first PR for opnsense. Now I use  "reconfigure" which is much cleaner and checks it.